pebblo/app/config/config_validation.py

```diff
@@ -42,6 +42,45 @@ def validate(self):
             )
 
 
+class StorageConfig(ConfigValidator):
+    def validate(self):
+        storage_type = self.config.get("type")
+        valid_storage_types = ["file", "db"]
+        if storage_type not in valid_storage_types:
+            self.errors.append(
+                f"Error: Unsupported storage type '{storage_type}' specified in the configuration."
+                f"Valid values are {valid_storage_types}"
+            )
+
+        # Set deprecated warning message for file storage type
+        if storage_type == "file":
+            deprecate_error = "DeprecationWarning: 'File' Storage Type will be deprecated starting from Pebblo version 0.0.19, use 'DB' instead"
+            print(deprecate_error)
```

Comment:
Is this print valid or we should do a log ?

Reply:
Instead of using strung values can we use Enums which we have already defined

Reply:
Regarding print - when this config validation happens, logger is not initialised, so we used print here.